### PR TITLE
Fix bug 1216825 - Added share links

### PR DESCRIPTION
**Pull request by lcamacho**

### CODE REVIEW

**Comment by lcamacho** [comment]
@peterbe r?

File: airmozilla/main/templates/main/event.html [hunk 1]
```diff
@@ -334,6 +334,22 @@ <h1 class="entry-title">
         </div>
       {% endif %}
 
+      <div class="tab">
```
Comment (peterbe) [inline airmozilla/main/templates/main/event.html]: This whole thing need to be wrapped in a `{% if event.is_public() %}` block.

**Comment by peterbe** [comment]
It looks a bit weird in a new tab since it's meant to be a pop-up
screenshot 2015-10-22 12 02 46

Perhaps we should consider this: http://jsfiddle.net/stichoza/eyxtj/

**Comment by peterbe** [comment]
Actually, the comment above is applicable to all three options.

**Comment by lcamacho** [comment]
@peterbe this new version should work

File: airmozilla/main/templates/main/event.html [hunk 2]
```diff
@@ -334,6 +335,24 @@ <h1 class="entry-title">
         </div>
       {% endif %}
 
+      {% if event.is_public() %}
+        <div class="tab">
+          <input type="radio" id="tab-share" name="tab-group-1">
+          <label for="tab-share">Share</label>
+          <div class="content">
+            <div>
+              <a class="share" href="https://www.facebook.com/sharer/sharer.php?u={{ make_absolute(url('main:event', event.slug)) }}" target="_blank">Share on Facebook</a>
```
Comment (peterbe) [inline airmozilla/main/templates/main/event.html]: The URL should be `urlencode`'ed too. Most characters in it are safe but it's best practice to URL encode them too.

File: airmozilla/main/templates/main/event.html [hunk 3]
```diff
@@ -334,6 +335,24 @@ <h1 class="entry-title">
         </div>
       {% endif %}
 
+      {% if event.is_public() %}
+        <div class="tab">
+          <input type="radio" id="tab-share" name="tab-group-1">
+          <label for="tab-share">Share</label>
+          <div class="content">
+            <div>
```
Comment (peterbe) [inline airmozilla/main/templates/main/event.html]: Why use individual `<div>` tags and not `<p>`? The paragraph tag feels more semantic. Is there a particular reason why `<div>` is better?

Comment (lcamacho) [inline airmozilla/main/templates/main/event.html]: ``<p>`` left too much space between links nothing else

File: airmozilla/main/templates/main/event.html (hunk 2 above)
Comment (lcamacho) [inline airmozilla/main/templates/main/event.html]: From here https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#url any special character should be encoded

File: airmozilla/main/templates/main/event.html (hunk 3 above)
Comment (lcamacho) [inline airmozilla/main/templates/main/event.html]: I mean this
captura de pantalla 2015-10-22 a las 16 34 35

File: airmozilla/main/templates/main/event.html (hunk 2 above)
Comment (peterbe) [inline airmozilla/main/templates/main/event.html]: We're not using that built-in and the `url()` function will return something like '/foo' which is fine. The the `make_absolute_url('/foo')` will return `http://localhost:8000/foo` which is also fine but that means that the final HTML becomes `href="https://www.facebook.com/sharer/sharer.php?u=http://localhost:8000/foo"` when instead it should be `href="https://www.facebook.com/sharer/sharer.php?u=http%3A%2F%2Flocalhost%3A8000%2Ffoo"`

File: airmozilla/main/templates/main/event.html (hunk 3 above)
Comment (peterbe) [inline airmozilla/main/templates/main/event.html]: that's probably better. The day we have a 4th or 5th or Nth link, we'll have to consider changing to...
```
<p>
    <a ...>...</a><br>
    <a ...>...</a><br>
    <a ...>...</a><br>

    <a ...>...</a><br>
    <a ...>...</a><br>
    <a ...>...</a><br>
</p>
```

**Comment by lcamacho** [comment]
@peterbe r?

**Comment by peterbe** [comment]
THANK YOU AGAIN!

**Pull request merged**